tests/test_pickup.py

```diff
@@ -1,148 +1,56 @@
-# Unit tests related to 'Pickups' (https://www.easypost.com/docs/api#pickups).
-
-import datetime
-import time
-
 import pytest
-import pytz
 
 import easypost
 
-ONE_DAY = datetime.timedelta(days=1)
 
+def generate_pickup(one_call_buy_shipment, basic_pickup):
```

Comment:
Let's call this `create_pickup` instead, it more clearly defines what's happening.